e107_languages/English/admin/lan_admin.php

```diff
@@ -201,6 +201,8 @@
 define("ADLAN_187", "Unable to create [x]. Please check your folder permissions.");
 define("ADLAN_188", "Your timezone setting [x] is invalid. It has been reset to UTC. To Modify, please go to Admin -> Preferences -> Date Display Options.");
 define("ADLAN_189", "The following plugins are not compatible with this version of e107 and should be uninstalled: ");
+define("ADLAN_190", "Need Help?");
+define("ADLAN_191", "Connect with our community for <a href='https://e107help.org' rel='external'>free support</a> with any e107 issues you may encounter.");
```

Comment:
Thanks. There should be no HTML in LANS. 
`Connect with our community for [free support] with any e107 issues you may encounter.`
..should be the value. 
And then in the code: 
`$message = str_replace(array('[',']'), array("<a href='https://e107help.org' rel='external'>","</a>"), ADLAN_191);`

Reply:
Tested this and  [free support]  not work for me as link.

Reply:
@yesszus Did you change ADLAN_191 ?

Reply:
Yes!